### PR TITLE
Documentation for core concepts

### PR DESCRIPTION
**Pull request by Bhumi101097**
Documentation for core concepts

### CODE REVIEW

File: website/docs/core-concepts/events.md [hunk 1]
```diff
@@ -4,4 +4,59 @@ sidebar_position: 9
 
 # Events
 
-## Coming Soon
+Events define when to trigger a notification for a webhook.
+
+Webhooks
+Webhooks are used to notify the users when defined events have occurred. These allow Dega to send POST requests to user-configured URLs to send them a notification about it. The request contains a triggered event and the result could be a notification on the set URL. 
+To use webhooks, setting up Hukz is required. Hukz is a simple & lightweight service implemented in GO to add webhooks to your application. This service is intended to fire webhooks on different events that are registered.
```
Comment (deshetti) [inline website/docs/core-concepts/events.md]: Link to Hukz repo

File: website/docs/core-concepts/claims.md [hunk 2]
```diff
@@ -4,4 +4,28 @@ sidebar_position: 3
...
+- Rating - This allows you to rate the claim, i.e True, False, etc.
+- Claim Date - Date on which the claim was made
+- CheckedDate - Date on which the claim was checked by fact-checkers
+- Description - Describes what the claim is for.
+- Claim Sources - This allows you to add a list of claim sources.
+    By clicking on the Add Claim Sources button, new fields are added to add information about the claim sources. Url and description can be provided for each claim source.
+- Review Sources - This allows you to add a list of review sources.
+    By clicking on the Add Review Sources button, new fields are added to add information about the review sources. Url and description can be provided for each review source.
+
+Apart from the basic information, extra data can be added such as SEO metadata. SEO metadata includes title, description, and canonical URL. It can be added by clicking on the Add MetaData button.
```
Comment (deshetti) [inline website/docs/core-concepts/claims.md]: Add a page for `Search Engine Optimisation` in `Features` and link to that. That page can have more details on how and why this SEO Metadata is helpful for SEO.

Add this as a sub-section with the title `SEO Metadata`. Do the same across all the docs where applicable.

File: website/docs/core-concepts/claims.md [hunk 3]
```diff
@@ -4,4 +4,28 @@ sidebar_position: 3
...
+
+There is also a provision to add more information about the claim if the field is not included in the form. The user can use the Meta fields section by clicking on the Meta Fields section to feed in extra data as a JSON object.
```
Comment (deshetti) [inline website/docs/core-concepts/claims.md]: Add a page for `Extending existing features` in `Features` and link to that. That page can have more details on how  Metafields can be used to capture any additional information to extend the OOB functionality of a certain feature. Would be helpful to add an example like https://health.factly.in/ on how we used Metafields to build the homepage.

Add this as a sub-section with the title `Metafields`. Do the same across all the docs where applicable.

File: website/docs/core-concepts/fact-checks.md [hunk 4]
```diff
@@ -4,4 +4,49 @@ sidebar_position: 2
 
 # Fact Checks
 
-## Coming Soon!
\ No newline at end of file
+Fact-checks are posts that provide information about a verified claim made by a claimant with descriptions and facts about the claim using textual content and images.
```
Comment (deshetti) [inline website/docs/core-concepts/fact-checks.md]: Fact-checks are posts of format: `fact-check`. Fact-checks have all the features available in a regular post (format: `default`) and are extended to add features specific to fact-checking.

File: website/docs/core-concepts/fact-checks.md [hunk 5]
```diff
@@ -4,4 +4,49 @@ sidebar_position: 2
...
+Fact-Check posts can include claims and images.
+New fact-check can be created from the Fact-Checks menu by clicking on the New Fact-Check button on the right side.
+
+The user can start writing fact-check using an editor which is a rich text editor. It makes embedding images, links, or any related content very easy.
+
+The user can directly publish or can save it as a draft. The user can also mark the post as ready to Publish.
+ 
+Each post created has status from any of the following.
```
Comment (deshetti) [inline website/docs/core-concepts/fact-checks.md:15]: Change `post` to `fact-check` wherever applicable in the doc. It is a bit confusing to talk in terms of `post` while discussing `fact-check`.

File: website/docs/core-concepts/fact-checks.md [hunk 6]
```diff
@@ -4,4 +4,49 @@ sidebar_position: 2
...
+- Draft
+- Published
+- Ready to Publish
+
+To save as draft click on the Save button on the right and to save as Ready to Publish; before clicking on the Save button make sure Ready to Publish is checked.
+To publish the Fact-Check click on the Publish button. Only the user who is either admin or an editor can publish the fact-check.
```
Comment (deshetti) [inline website/docs/core-concepts/fact-checks.md]: These are only default policies, right? Change this accordingly that policies can be defined for different actions. In fact, we can add two new sections across all the entities:

1. `Policies`: Section explaining the default policies on that entity and how the users can extend it by creating custom policies. Linkt to the `policy` page.
2. `Events`: Section explaining the possible events on that entity and how they could configure them. Link to the `events` page.

File: website/docs/core-concepts/events.md [hunk 7]
```diff
@@ -4,4 +4,59 @@ sidebar_position: 9
 
 # Events
 
-## Coming Soon
+Events define when to trigger a notification for a webhook.
```
Comment (deshetti) [inline website/docs/core-concepts/events.md]: Add examples on how and when you could trigger events come in handy:

- [ ] A webhook to trigger a static site to build based on updates
- [ ] Notifications on Slack & Google Chat when a certain action is made

Add following pages on `How-to Guides`
- [ ] Trigger build on Static Sites. Use Netlify, Render, Vercel as sub-sections. Check with @fyrepenguin for details
- [ ] Example for notification on Slack
- [ ] Example for notification on Google Chat

File: website/docs/core-concepts/formats.md [hunk 8]
```diff
@@ -4,4 +4,23 @@ sidebar_position: 5
 
 # Formats
 
-## Coming Soon!
\ No newline at end of file
+Formats for a space can be created from the Formats menu. Default formats for Dega can be added by clicking on the Create default Formats.
```
Comment (deshetti) [inline website/docs/core-concepts/formats.md]: Mention that it is a `required` step before working on a `Post`.

File: website/docs/core-concepts/formats.md [hunk 9]
```diff
@@ -4,4 +4,23 @@ sidebar_position: 5
 
 # Formats
 
-## Coming Soon!
\ No newline at end of file
+Formats for a space can be created from the Formats menu. Default formats for Dega can be added by clicking on the Create default Formats. 
+
+The default formats for Dega are
```
Comment (deshetti) [inline website/docs/core-concepts/formats.md]: The *current* default formats available in Dega are:

File: website/docs/core-concepts/formats.md [hunk 10]
```diff
@@ -4,4 +4,23 @@ sidebar_position: 5
...
+- Fact-Check
+- Article
+
+Fact-Check and Article formats are mandatory formats for Dega.
+
+- Name: This allows the user to add a Name of the format
+- Featured: Check if the claimant is featured
```
Comment (deshetti) [inline website/docs/core-concepts/formats.md]: This field can be used to `highlight` the `Claimant` on the website if `featured` is enabled.

File: website/docs/core-concepts/media.md [hunk 11]
```diff
@@ -4,4 +4,21 @@ sidebar_position: 7
 
 # Media
 
-## Coming Soon!
+Media to be used across the space can be found here. Media uploaded from any other place across Dega is also listed here.
```
Comment (deshetti) [inline website/docs/core-concepts/media.md]: Change this line to:

All the media files used across the space are available here. The media files used in the space are reusable across different posts, settings, etc.

File: website/docs/core-concepts/media.md [hunk 12]
```diff
@@ -4,4 +4,21 @@ sidebar_position: 7
...
+Admin of an organization can limit the number of media for a space, which can be changed from the Space Requests menu by making a request to change the number of media for a space.
```
Comment (deshetti) [inline website/docs/core-concepts/media.md]: Change to: 

The administrator of an organization can limit the number of media files allowed within a space. This limit can be configured from the `Space Requests` menu by creating a request to change the media file limit for a space.

File: website/docs/core-concepts/organisations.md [hunk 13]
```diff
@@ -4,4 +4,11 @@ sidebar_position: 8
 
 # Organisations
 
-## Coming Soon!
+To access Dega, creating an organization is mandatory. If Super Organisation is created while installing Dega, then the user can access Dega or else needs to create an organization from Kavach for having an access to Dega.
```
Comment (deshetti) [inline website/docs/core-concepts/organisations.md]: Change to:

To access Dega, creating an organization is mandatory. The first organisation, created will be a `Super Organisation`. While a `Super Organisation` could be used as a normal organisation within a Dega instance, it has the following capabilities to manage the entire dega instance: 

- The users within the `Super Organisation` will have access to all the other organisations and their spaces. This is for support staff to help other organisations with any support queries.
- The admins within the `Super Organisation` will have admin access to the entire instance and will be able to manage the rest of the organisations and spaces. They have the ability to manage the usage limits for an oraganisation or spaces within.

If Super Organisation is created while installing Dega, the user can access Dega or else needs to create an organization from Kavach for having an access to Dega.

File: website/docs/core-concepts/organisations.md [hunk 14]
```diff
@@ -4,4 +4,11 @@ sidebar_position: 8
...
+There is only one Super Organisation for a Dega instance but can create multiple Organisations as per user requirements. Applications can be added to an organization.
```
Comment (deshetti) [inline website/docs/core-concepts/organisations.md]: Change the rest of the page to be more descriptive and check on the grammar. For eg: _Applications can be added to an organization._ seems like a random statement if no context is provided on what an `application` is.

File: website/docs/core-concepts/policies.md [hunk 15]
```diff
@@ -4,4 +4,23 @@ sidebar_position: 9
 
 # Policies
 
-## Coming Soon
+Policies are mandatory for Dega to work as expected. Default Policies can be added by clicking on Create Default Policies.
```
Comment (deshetti) [inline website/docs/core-concepts/policies.md:7]: Add section like on the rest of the entities about who/how can one access/edit policy. Meaning, policy to manage policy.

File: website/docs/core-concepts/posts.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+To save as draft click on the Save button on the right and to save as Ready to Publish, before clicking on the Save button make sure Ready to Publish is checked.
+
+To publish the Post click on the Publish button. Only the user who is either admin or an editor can publish the Post.
+
+Once the post is created, templates can be created from the post which can be used for future posts. Templates can be created by clicking on the Create Template button.
+New posts can be created from templates by clicking on any of the templates available from the Templates section which is at the top of the Post page
+
+The user can add information about the post by clicking on the Settings button.
+
+- Featured Image - Image to be set as thumbnail for the post
```
Comment (deshetti) [inline website/docs/core-concepts/posts.md]: not just thumbnail, a featured image is more than that. check this on wp for eg: https://wordpress.com/support/featured-images

File: website/docs/core-concepts/posts.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+- Excerpt - Brief description of the post
+- Tags - This allows you to add or assign tags to the post
```
Comment (deshetti) [inline website/docs/core-concepts/posts.md]: link to tags, categories, authors etc

File: website/docs/core-concepts/posts.md [hunk 18]
```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 2
+---
+
+# Posts 
+
+Posts are articles or stories which may include images or other illustrations.
+New Post can be created from the Posts menu by clicking on the New Post button on the right side.
+
+The user can start writing a post using an editor which is a rich text editor. It makes embedding images, links, or any related content very easy.
```
Comment (deshetti) [inline website/docs/core-concepts/posts.md]: Create a page for `Powerful Editor` in `Features` and link to that

**Review by deshetti — COMMENTED** [review]
- [ ] Link to related pages in the docs where applicable
- [ ] Add examples where applicable
- [ ] Add sub-sections across all the entities:
  - [ ] Policies: Section explaining the default policies on that entity and how the users can extend it by creating custom policies. Linkt to the policy page.
  - [ ] Events: Section explaining the possible events on that entity and how they could configure them. Link to the events page.
  - [ ] SEO Metadata
  - [ ] Metafields
- [ ] Add following pages on `How-to Guides`
  - [ ] Trigger build on Static Sites. Use Netlify, Render, Vercel as sub-sections. Check with @fyrepenguin for details
  - [ ] Example for notification on Slack
  - [ ] Example for notification on Google Chat
- [ ] Add following pages on `Features`
	- [ ] Search Engine Optimisation
	- [ ] Extending existing features
	- [ ] Powerful Editor
	- [ ] Search & Filtering
	    - This would be about how we implemented a powerful relevant and typo tolerant search and filtering using Meilisearch. Look through Meilisearch docs for any relevant info to add here.
    - [ ] Proxy server for rich media
	    -   Built-in support for oembed proxy server for rich media using  [Iframely](https://github.com/itteco/iframely). Look through iframely docs for any relevant info to be add here.
    - [ ] Multitenancy
	  - Discuss support for multiple organisations and spaces within
  - [ ] Event Notifications
	  - Discuss built-in support for event notifications using webhooks. 
- [ ] In Posts, Fact checks: Add about schemas that are generated automatically

File: website/docs/core-concepts/events.md (hunk 1 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/events.md]: Updated

File: website/docs/core-concepts/fact-checks.md (hunk 6 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/fact-checks.md]: Updated

File: website/docs/core-concepts/fact-checks.md (hunk 5 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/fact-checks.md:15]: updated

File: website/docs/core-concepts/formats.md (hunk 9 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/formats.md]: updated

File: website/docs/core-concepts/posts.md (hunk 17 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/posts.md]: updated

File: website/docs/core-concepts/media.md (hunk 12 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/media.md]: updated

File: website/docs/core-concepts/media.md (hunk 11 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/media.md]: updated

File: website/docs/core-concepts/formats.md (hunk 10 above)
Comment (Bhumi101097) [reply website/docs/core-concepts/formats.md]: updated

**Review by deshetti — APPROVED** [review]

**Pull request merged**